src/views/Give/Give.scss

```diff
@@ -546,6 +546,7 @@
 
   .give-info {
     width: 220px;
```

Comment:
this width could also be removed? unless it's intentional

Reply:
Due to the way the CSS was coded at the time, it's probably intentional.